scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2470,7 +2470,7 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
                 case name: Term.Placeholder =>
                   Some(
                     atPos(tree, tree)(
-                      Term.Param(Nil, autoPos(Name.Anonymous()), None, None)
+                      Term.Param(Nil, atPos(name, name)(Name.Anonymous()), None, None)
```

Comment:
is this case tested? i thought this was the one that was failing for me originally, i don't see it in new tests.